### PR TITLE
Simplify and update CBS.xml, fix #12526

### LINKED ISSUE #12526: CBS.xml and CBSNews.xml have conflicting rules
CBSNews.xml added in https://github.com/EFForg/https-everywhere/pull/11933 seems to conflict with CBS.xml which has:

```xml
	<rule from="^http://markets\.cbsnews\.com/$"
		to="https://markets.financialcontent.com/cbsnews" />

	<rule from="^http://markets\.cbsnews\.com/"
		to="https://markets.financialcontent.com/" />
```

As far as I can tell, CBS.xml's rewrite rules are incorrect in this case, and CBSNews' `markets.cbsnews.com` should always take effect.

(also, seems like a good idea to somehow auto-test that different rulesets don't match same URLs)

### PR DESCRIPTION
**Pull request by cschanaj**
Fix #12526

### CODE REVIEW

File: src/chrome/content/rules/CBS-Sports-Network.xml [hunk 1]
```diff
@@ -1,24 +1,11 @@
...
 	For other CBS coverage, see CBS.xml.
-
-
-	CN: *.cstv.com
-
 -->
-<ruleset name="CBS Sports Network " default_off="mismatched">
-
-	<!--	Akamai	-->
+<ruleset name="CBS Sports Network">
```
Comment (J0WI) [inline src/chrome/content/rules/CBS-Sports-Network.xml]:
```
affiliates.cbssportsnetwork.com
dev.cbssportsnetwork.com

File: src/chrome/content/rules/CBSStatic.com.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!--
+	Non-functional hosts
+		SSL peer certificate was not OK:
```
Comment (J0WI) [inline src/chrome/content/rules/CBSStatic.com.xml]: Can we redirect them to the `-secure` variant?

File: src/chrome/content/rules/CBS.xml [hunk 3]
```diff
@@ -14,152 +11,23 @@ Fetch error: http://dw.com.com/ => https://dw.com.com/: (28, 'Connection timed o
...
-			- a868.g.akamai.net
-
-				- a248.e.akamai.net/cnwk.1d/
-				- a248.e.akamai.net/=/868/1957/7d/img.com.com/
-
-	cnwk.1d/i/
-	/=/5048/1957/7d/img.com.com/i/
-
-
-	Nonfunctional domains:
```
Comment (J0WI) [inline src/chrome/content/rules/CBS.xml]: Please double check the `*.cbs.com` hosts and keep existing ones (e.g. `download.cnet.com`)

**Comment by cschanaj** [comment]
@J0WI I have improved the coverage according to the old comments. thanks.

File: src/chrome/content/rules/CBSInteractive.com.au.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!--
```
Comment (J0WI) [inline src/chrome/content/rules/CBSInteractive.com.au.xml]: I only found one more here:
`asia.cbsinteractive.com`
so let's complete it.

File: src/chrome/content/rules/FindArticles.com.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<!--
```
Comment (J0WI) [inline src/chrome/content/rules/FindArticles.com.xml]: I only found two more here:
```
es.findarticles.com
mx.findarticles.com 
```
so let's complete it.

**Comment by J0WI** [comment]
Please make sure that you do not overwrite existing commits in this branch.

**Comment by cschanaj** [comment]
I have covered nearly all domain listed in the old non-function domain list, except `*.cstv.com`, `*.ocsn.com`. Besides, `*.cbs.dk` and `*.zol.com.cn` do not seem to be related to the CBS and thus I have omitted them. `*.cnet.com` is covered in [`CNET.com.xml`](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/CNET.com.xml). I hope this will address https://github.com/EFForg/https-everywhere/pull/15965#discussion_r212819886.

**Comment by J0WI** [comment]
> `*.cnet.com` is covered in `CNET.com.xml`. I hope this will address #15965 (comment).

Oops, I missed this file, sorry!
But there are some great new rules now, they all LGTM.

**Pull request merged**